eng/build.ps1

```diff
@@ -20,46 +20,78 @@ Param(
+  Write-Host "                                 [Default: Your machine's architecture.]"
+  Write-Host "  -binaryLog (-bl)               Output binary log."
+  Write-Host "  -configuration (-c)            Build configuration: Debug, Release or [CoreCLR]Checked."
+  Write-Host "                                 Pass a comma-separated list to build for multiple configurations."
+  Write-Host "                                 [Default: Debug]"
+  Write-Host "  -help (-h)                     Print help and exit."
+  Write-Host "  -librariesConfiguration (-lc)  Libraries build configuration: Debug or Release."
+  Write-Host "                                 [Default: Debug]"
+  Write-Host "  -os                            Build operating system: Windows_NT, Linux, OSX, or Browser."
+  Write-Host "                                 [Default: Your machine's OS.]"
+  Write-Host "  -runtimeConfiguration (-rc)    Runtime build configuration: Debug, Release or [CoreCLR]Checked."
+  Write-Host "                                 [Default: Debug]"
+  Write-Host "  -subset (-s)                   Build a subset, print available subsets with -subset help."
+  Write-Host "                                 '-subset' can be omitted if the subset is given as the first argument."
+  Write-Host "                                 [Default: Builds the entire repo.]"
+  Write-Host "  -verbosity (-v)                MSBuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic]."
+  Write-Host "                                 [Default: Minimal]"
+  Write-Host "  -vs                            Open the solution with VS using the locally acquired SDK. Path or solution name."
+  Write-Host "                                 (Example: -vs Microsoft.CSharp)"
   Write-Host ""
 
   Write-Host "Actions (defaults to -restore -build):"
-  Write-Host "  -restore                Restore dependencies"
-  Write-Host "  -build                  Build all source projects (short: -b)"
-  Write-Host "  -rebuild                Rebuild all source projects"
-  Write-Host "  -test                   Build and run tests (short: -t)"
-  Write-Host "  -pack                   Package build outputs into NuGet packages"
-  Write-Host "  -sign                   Sign build outputs"
-  Write-Host "  -publish                Publish artifacts (e.g. symbols)"
-  Write-Host "  -clean                  Clean the solution"
+  Write-Host "  -build (-b)             Build all source projects."
+  Write-Host "  -clean                  Clean the solution."
+  Write-Host "  -pack                   Package build outputs into NuGet packages."
+  Write-Host "  -publish                Publish artifacts (e.g. symbols)."
+  Write-Host "  -rebuild                Rebuild all source projects."
+  Write-Host "  -restore                Restore dependencies."
+  Write-Host "  -sign                   Sign build outputs."
+  Write-Host "  -test (-t)              Build and run tests."
```

Comment:
Does this actually build tests, or just run? 
Also, does it build the library under test, or just the test library?

good to note this.

Reply:
That's an interesting observation and something worth including. I believe it just builds the test library, but I've not used it much so I'm not sure. Can you clarify this @janvorli?

Reply:
the target test just runs the tests. In order to build you need to do `-build` and passing the right subset that includes tests.

Reply:
The `-test` switch incrementally restores and builds the test projects. You can specify `-test -testnobuild` to only run the tests. https://github.com/dotnet/runtime/blob/617692461d55b7ae8162138e572f280e4c1df5f5/eng/build.ps1#L108-L113

Reply:
Thanks for clarifying!